Content.Server/Roles/Job.cs

```diff
@@ -39,24 +39,13 @@ public override void Greet()
             if (Mind.TryGetSession(out var session))
             {
                 var chatMgr = IoCManager.Resolve<IChatManager>();
-                var chatSys = IoCManager.Resolve<IEntitySystemManager>().GetEntitySystem<ChatSystem>();
-                chatMgr.DispatchServerMessage(session, Loc.GetString("job-greet-introduce-job-name", ("jobName", Name)));
+                chatMgr.DispatchServerMessage(session, Loc.GetString("job-greet-introduce-job-name",
+                    ("jobName", CultureInfo.CurrentCulture.TextInfo.ToTitleCase(Name))));
 
                 if(Prototype.RequireAdminNotify)
                     chatMgr.DispatchServerMessage(session, Loc.GetString("job-greet-important-disconnect-admin-notify"));
 
                 chatMgr.DispatchServerMessage(session, Loc.GetString("job-greet-supervisors-warning", ("jobName", Name), ("supervisors", Loc.GetString(Prototype.Supervisors))));
-
-                if(Prototype.JoinNotifyCrew && Mind.CharacterName != null)
-                {
-                    if (Mind.OwnedEntity != null)
-                    {
-                        chatSys.DispatchStationAnnouncement(Mind.OwnedEntity.Value,
-                            Loc.GetString("job-greet-join-notify-crew", ("jobName", Name),
-                                ("characterName", Mind.CharacterName)),
-                            Loc.GetString("job-greet-join-notify-crew-announcer"), false);
-                    }
-                }
```

Comment:
This never run because normally when Greet() is called you aren't actually spawned yet, so the Mind has ho OwnedEntity.